miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/TagUtils.java

```diff
@@ -1,6 +1,9 @@
 package uk.ac.bbsrc.tgac.miso.webapp.util.form;
 
 import org.apache.log4j.Logger;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
 
 public class TagUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)